warehouse/admin/templates/admin/sponsors/edit.html

```diff
@@ -136,6 +136,25 @@ <h3 class="box-title">Create sponsor</h3>
             {{ render_field("One Time", form.one_time, "sponsor-one-time") }}
             {{ render_field("Sidebar", form.sidebar, "sponsor-sidebar") }}
 
+            {% if sponsor %}
+            <div class="box-header with-border">
+              <h3 class="box-title">Info from Python.org Logo Placement API</h3>
+              <hr>
+
+              <div class="">
+                <label for="{{ input_id }}" class="col-sm-2 align-right">Origin</label>
+                <p>{{ sponsor.origin }}</p>
+                <label for="{{ input_id }}" class="col-sm-2 align-right">Level Name</label>
+                <p>{{ sponsor.level_name|default("---", true) }}</p>
+                <label for="{{ input_id }}" class="col-sm-2 align-right">Level Order</label>
+                <p>{{ sponsor.level_order|default("---", true) }}</p>
+                <label for="{{ input_id }}" class="col-sm-2 align-right">Slug</label>
```

Comment:
What do labels mean without a form?  Also they all have the same `for` attribute which doesn’t make sense to me

Reply:
But the real issue here is probably the use of labels outside of a form!